cmd/tracee-rules/internal/config/config.go

```diff
@@ -0,0 +1,89 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+type TraceeRulesCliConfig struct {
+	RegoConfig struct {
+		Aio               bool
+		EnableParsedEvent bool
+		PartialEval       bool
+		RunTimeTarget     string
+	}
+	Rules struct {
+		InputDirectory string
+		RuleIds        []string
+	}
+	Webhook struct {
+		Url         string
+		Template    string
+		ContentType string
+	}
+	Input struct {
+		File   string
+		Format string
+	}
+	Output struct {
+		Template string
+	}
+	Pprof struct {
+		Enable  bool
+		Address string
+	}
+	List bool
+}
+
+var Config TraceeRulesCliConfig
```

Comment:
same as above, let's not use a global var, instead LoadConfig should return the configuration

Reply:
Same as before, i'll change it to a return